x/bank/internal/types/tags.go

```diff
@@ -1,4 +1,4 @@
-package tags
```

Comment:
ditto, not consistent. Unless we come to consensus for this on #4438

Reply:
I'll update #4438 accordingly